### PR TITLE
adding reference on front page to radiuss external ci

### LINKED ISSUE #11: Add a reference to Radiuss-Shared-CI
## Add Reference to:

RADIUSS developed a shared GitLab CI implementation that lets you get started very quickly and coherently with other projects.

### Intended primarily to:
Open-source RADIUSS projects hosted on GitHub and willing to run extra CI on LC GitLab.

### Prerequisites
- support of Spack packaging.
- CMake build system (strongly recommended, but not mandatory).

### Documentation
https://radiuss-shared-ci.readthedocs.io/en/latest/

### Sources
https://github.com/LLNL/radiuss-shared-ci

### LINKED ISSUE #12: Add reference to MFEM Github Actions
## Add reference to

RADIUSS developed a set of Github Actions to install MFEM dependencies and MFEM itself in the Github Actions context.

### Primarily intended to

Open-source projects hosted on Github and willing to install the already supported projects or contribute new projects to the repo (we could extend that to other math projects to build a coherent set of actions).

### Sources

https://github.com/mfem/github-actions

### Notes

Not building any containers, builds dependencies on top of classic runners images. See MFEM/mfem actions for usage example.

### PR DESCRIPTION
**Pull request by vsoch**
This will close #11 and will close #12

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

### CODE REVIEW

**Comment by vsoch** [comment]
Preview: https://output.circle-artifacts.com/output/job/bf7c17c1-4367-4c53-a2ee-a37e2b95fd2d/artifacts/0/ci/index.html

image

**Comment by vsoch** [comment]
And now with added MFEM reference:

https://output.circle-artifacts.com/output/job/a1f8c840-5b08-43d3-a7cf-5983c568ace3/artifacts/0/ci/index.html

image

I decided to add just a link because maintaining tutorial or other docs would be dangerous, maintainership wise. It would get out of date and then... not work!

File: index.md [hunk 1]
```diff
@@ -13,3 +13,11 @@ Welcome to the RADIUSS Continuous Integration (CI) set of instructions. RADIUSS
 Explore the different build frameworks on the left to learn how to generate your
 CI pipeline. If you have a framework that is not listed here, please <a href="https://github.com/rse-ops/ci/issues" target="_blank">let us know!</a>
 
+<br>
+
+## Supported Tooling
+
+The following external documentation sites can additionally provide support to test RADIUSS-specific projects.
+
+ - [RADIUSS Shared CI](https://radiuss-shared-ci.readthedocs.io/en/latest/) is intended to provide open source RADIUSS projects with the ability to be hosted on GitHub, but run extra CI on LC GitLab. To use this setup you will need a spack package for your software. The Cmake build system is strongly recommended but not mandatory.
```
Comment (adrienbernede) [inline index.md:22]: Cmake -> CMake

**Review by adrienbernede — APPROVED** [review]
LGTM

File: index.md (hunk 1 above)
Comment (vsoch) [inline index.md:22]:
```suggestion
 - [RADIUSS Shared CI](https://radiuss-shared-ci.readthedocs.io/en/latest/) is intended to provide open source RADIUSS projects with the ability to be hosted on GitHub, but run extra CI on LC GitLab. To use this setup you will need a spack package for your software. The CMake build system is strongly recommended but not mandatory.
```

**Comment by adrienbernede** [comment]
Thanks 👍 !

**Pull request merged**